### ISSUE TITLE
NVDA reports selection incorectly in VS 2008 code editor

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1358]
**Reported by aleksey_s on 2011-02-08 08:37**
STR:
- open vs 2008
- file->new->file
- choose a c++ file template
- type something like

```
#include <iostream>
```
- press "enter" to create new line
- navigate up to place cursor on the first line
- press shift+end
  Actual results:
  NVDA announces "#include <iostream"
  Expected results:
  NVDA should announce "#include <iostream>"

**Comment by leonardder** [comment]
Closing as won't fix because VS2008 is quite old now
